avm/res/app/container-app/main.bicep

```
@@ -301,3 +299,121 @@ type roleAssignmentType = {
   @description('Optional. The Resource Id of the delegated managed identity resource.')
   delegatedManagedIdentityResourceId: string?
 }[]?
+
+type container = {
+  @description('Optional. Container start command arguments.')
+  args: string[]?
+
+  @description('Optional. Container start command.')
+  command: string[]?
+
+  @description('Optional. Container environment variables.')
+  env: environmentVar[]?
+
+  @description('Required. Container image tag.')
+  image: string
+
+  @description('Optional. Custom container name.')
+  name: string
+
+  @description('Optional. List of probes for the container')
+  probes: containerAppProbe[]?
+
+  @description('Required. Container resource requirements.')
+  resources: object
+
+  @description('Optional. Container volume mounts.')
+  volumeMounts: volumeMount[]?
+}
+
+type environmentVar = {
+  @description('Required. Environment variable name.')
+  name: string
+
+  @description('Optional. Name of the Container App secret from which to pull the environment variable value.')
+  secretRef: string?
+
+  @description('Optional. Non-secret environment variable value.')
+  value: string?
+}
+
+type containerAppProbe = {
+  @description('Optional. Minimum consecutive failures for the probe to be considered failed after having succeeded. Defaults to 3. Minimum value is 1. Maximum value is 10.')
+  failureThreshold: int?
+
+  @description('Optional. HTTPGet specifies the http request to perform.')
+  httpGet: containerAppProbeHttpGet?
+
+  @description('Optional. Number of seconds after the container has started before liveness probes are initiated. Minimum value is 1. Maximum value is 60')
```

```suggestion
  @description('Optional. Number of seconds after the container has started before liveness probes are initiated.')
```

Same for other parameters. With min/max defined in the metadata you don't need it in the description. I'd recommend to go with the metadata consistently and updat the descriptions accordingly. For example, `failureThreshold` is not using it though it probably should.